src/main/java/org/kaazing/github/Action.java

```diff
@@ -194,4 +211,168 @@ public String getDescription() {
         }
     };
 
+    public static Action LIST_ISSUES_CLOSED_WITHIN_DAY = new Action() {
+
+        @Override
+        public void accept(GHRepository t) {
+        	List<GHIssue> closed = null;
+        	try {
+        		closed = t.getIssues(GHIssueState.CLOSED);
+        	} catch (IOException e) {
+        		throw new RuntimeException(e);
+        	}
+
+            Iterator<GHIssue> iter = closed.iterator();
+            while (iter.hasNext()) {
+                GHIssue next = iter.next();
+                FilterIssues filter = FilterIssues.CLOSED_WITHIN_DAYS;
+                String[] args = {"1"};
+                filter.setArgs(args);
+                if (filter.test(next))
+                	System.out.println(t.getFullName() + "#" + next.getNumber() + " -- " + next.getTitle());
+            }
+        }
+
+		@Override
+		public String getName() {
+			return "list-issues-recently-closed";
+		}
+
+		@Override
+		public String getDescription() {
+			return "get all recently closed issues";
+		}
+
+		@Override
+		public String listArgs() {
+			return "";
+		}
+
+    };
+    
+    public static Action LIST_ISSUES_UPDATED_WITHIN_DAY = new Action() {
+
+        @Override
+        public void accept(GHRepository t) {
+        	List<GHIssue> closed = null;
+        	try {
+        		closed = t.getIssues(GHIssueState.OPEN);
```

Comment:
Minor: You can static import these states I believe to make the code more readable